docs/www/configuration.md

```diff
@@ -102,6 +102,27 @@ redpanda:
     address: "0.0.0.0"
     port: 33145
   
+  # Address, ports, and certificates for the redpanda REST API.
```

Comment:
This belongs down here: https://github.com/vectorizedio/redpanda/pull/1103/files#diff-5c28b437613240862a9804c667091855f52b4f57ea179d0a2e58a033fa13cbcfL348

Reply:
@BenPope Sorry I missed this. The pandaproxy config is shown also in the pandaproxy top-level config but with different values. Can you clarify it?